sdk/identity/Azure.Identity/src/IdentityClient.cs

```diff
@@ -152,6 +188,77 @@ private Request CreateClientSecretAuthRequest(string tenantId, string clientId,
             return request;
         }
 
+        private Request CreateClientCertificateAuthRequest(string tenantId, string clientId, X509Certificate2 clientCertficate, string[] scopes)
+        {
+            Request request = _pipeline.CreateRequest();
+
+            request.Method = HttpPipelineMethod.Post;
+
+            request.Headers.SetValue("Content-Type", "application/x-www-form-urlencoded");
```

Comment:
nit:  Is there a member in `Azure.Core` for form types?   If not, maybe we should consider adding it there to keep building that out.

Reply:
Agreed.  I've added and issue to track this, https://github.com/Azure/azure-sdk-for-net/issues/6646. I'll make this change in a separate PR since it's a change to Azure.Core and requires rebuilding everything.